RecoTracker/LSTCore/src/alpaka/NeuralNetworkWeights.h

```diff
@@ -3,14 +3,16 @@
 
 #include <alpaka/alpaka.hpp>
 
+#include "FWCore/Utilities/interface/HostDeviceConstant.h"
+
 namespace ALPAKA_ACCELERATOR_NAMESPACE::lst::t5dnn {
-  ALPAKA_STATIC_ACC_MEM_GLOBAL const float bias_layer1[32] = {
+  HOST_DEVICE_CONSTANT const float bias_layer1[32] = {
```

Comment:
Are the const keywords supposed to be here still?

Reply:
Removed for clarity I suppose, since you removed the constexpr keywords in the other file.

Reply:
OK.

`constexpr` needs to be removed (because it results in `constexpr constexpr`), `const` could stay (because it results in `constexpr const` which is implied but allowed).

Anyway, I've removed it.